exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java

```diff
@@ -51,7 +52,10 @@ class SerializerTest {
               AggregationTemporality.CUMULATIVE,
               Collections.singletonList(
                   ImmutableDoublePointData.create(
-                      1633947011000000000L, 1633950672000000000L, KP_VP_ATTR, 5))));
+                      1633947011000000000L,
+                      1633950672000000000L,
+                      Attributes.of(TYPE, "mcds"),
```

Comment:
Before all the metrics had the same name, meter, attributes, which is not representative of anything the SDK could actually output. So added this attribute to make a bit more realistic